lib/windows.js

```diff
@@ -116,10 +114,8 @@ function windows() {
 	const processFileNameSizeBuffer = ref.alloc('uint32', pathLengthChars);
 	// Write process file path to buffer
 	kernel32.QueryFullProcessImageNameW(processHandle, 0, processFileNameBuffer, processFileNameSizeBuffer);
-	// Remove null characters from buffer
-	const processFileNameBufferClean = ref.reinterpretUntilZeros(processFileNameBuffer, wchar.size);
 	// Get process file path as a string
-	const processPath = wchar.toString(processFileNameBufferClean);
+	const processPath = wchar.toString(processFileNameBuffer).replace(/\0+/, '');
```

Comment:
It would only clean at the end, right?

```suggestion
	const processPath = wchar.toString(processFileNameBuffer).replace(/\0+$/, '');
```